CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 - Allow `sockaddr_ll` size, as reported by the Linux kernel, to be smaller then it's definition
   (#[1395](https://github.com/nix-rust/nix/pull/1395))
+- Fix corrupted `sendmmsg` call
```

Comment:
Could you be more specific?  Something like "Fix spurious errors using `sendmmsg` with multiple and/or long cmsgs`?

Reply:
updated